benefits/tech_scheme.md

```diff
@@ -0,0 +1,19 @@
+# Tech-scheme
+
+We are signed up to Techscheme, part of Cyclescheme, which you can upon successful completion of your probation.
+
+Techscheme lets you get the latest tech worth up to £2,000. You then spread the cost with payments from your salary across 12 or 24 months and make a National Insurance saving of up to 12% too. 
```

Comment:
I find this explanation a little confusing. Do you get help paying for the entire item or just part of it? Does the payment come automatically out of your salary? How does it make a National Insurance saving?

Reply:
I've changed the wording - I hope this is not more confusing now.